src/runtime/containerd-shim-v2/wait.go

```diff
@@ -152,6 +154,7 @@ func watchOOMEvents(ctx context.Context, s *service) {
 				if isGRPCErrorCode(codes.NotFound, err) || err.Error() == "Dead agent" {
 					return
 				}
+				time.Sleep(defaultCheckInterval)
 				continue
```

Comment:
Why `continue` instead of `break` here? If `GetOOMEevent()` fails once, are we expecting it to succeed by retrying?

Reply:
I‘m not sure. Maybe except that it will not broken by temporary errors.

Reply:
Fair enough. At least we have a rate limiter now.